test/functional/cmdLineTests/gcRegressionTests/gcRegressionTests.xml

```diff
@@ -352,15 +352,25 @@
   <output regex="no" type="required">&lt;/verbosegc&gt;</output>
   <output regex="no" type="success">All allocated blocks were freed.</output>
   <!-- allow memory leaks since the JIT leaks, unfortunately -->
+  <!-- Temporarily disable the usage of regex until the cmdlinetester is fixed.
+  Related: https://github.com/eclipse-openj9/openj9/issues/15235
   <output regex="yes" type="success">[0-9]* allocated blocks totaling [0-9]* bytes were not freed before shutdown!</output>
+  -->
+  <output regex="no" type="success">allocated blocks totaling</output>
```

Comment:
We should leave this part out, it seems a little more generic. Any success condition will allow the test to pass.

Reply:
Updated.